src/opnsense/mvc/app/views/OPNsense/IDS/index.volt

```diff
@@ -199,19 +199,30 @@ POSSIBILITY OF SUCH DAMAGE.
                 $.each(rows, function(key,uuid){
                     keyset.push(uuid);
                     if ( combine == undefined || keyset.length > combine) {
-                        deferreds.push(ajaxCall(url + keyset.join(',') +'/'+url_suffix, sendData={},null));
+                        (function (localKeyset) {
+                            promise = promise.then(function () {
+                                return ajaxCall(url + localKeyset.join(',') + '/' + url_suffix, sendData = {}, null);
+                            })
+                        }) (keyset);
```

Comment:
this function is probably not needed

Reply:
First I did without closure. All calls AjaxCall was with the last value of the keyset. I had to add this feature.

Reply:
I am talking about the outer function, not the promise